src/Debugger.jl

```diff
@@ -5,18 +5,14 @@ using Base.Meta: isexpr
 using REPL
 using REPL.LineEdit
 
-include("DebuggerFramework.jl")
-using .DebuggerFramework
-using .DebuggerFramework: FileLocInfo, BufferLocInfo, Suppressed
-
-using JuliaInterpreter: JuliaInterpreter, JuliaStackFrame, @lookup, Compiled, JuliaProgramCounter, JuliaFrameCode
+using JuliaInterpreter: JuliaInterpreter, JuliaStackFrame, @lookup, Compiled, JuliaProgramCounter, JuliaFrameCode,
+      finish!, enter_call_expr, step_expr!
 
 # TODO: Work on better API in JuliaInterpreter and rewrite Debugger.jl to use it
-using JuliaInterpreter: _make_stack, pc_expr,
-finish!, isassign, getlhs, do_assignment!, maybe_next_call!, enter_call_expr, is_call, step_expr!, _step_expr!,
-next_call!, iswrappercall, moduleof, next_line!, location
+# These are undocumented functions used by Debugger.jl from JuliaInterpreter
```

Comment:
Just a small nitpick — I think this would be more easily read like this:

    # These are undocumented functions from from JuliaInterpreter.jl used by Debugger.jl

Reply:
Thanks, updated.

Reply:
Oh shoot — I led you to introduce a typo 😳 `from from` should be `from`. Sorry!

Reply:
Submitted #17 to fix my blunder.